app/services/manageiq/providers/azure/number_of_vms_visibility_service.rb

```diff
@@ -0,0 +1,18 @@
+class ManageIQ::Providers::Azure::NumberOfVmsVisibilityService < ::NumberOfVmsVisibilityService
+  def determine_visibility(number_of_vms, platform)
+    @number_of_vms = number_of_vms
+
+    hash = super
+
+    # For Azure we will show two options in this case - private and public
+    if @number_of_vms > 1
+      hash[:hide].delete(:floating_ip_address)
+    end
+
+    hash
+  end
```

Comment:
@djberg96 - Where is this method called?

Reply:
It's used by the `MiqProvisionVirtWorkflow` class at https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_provision_virt_workflow.rb#L985